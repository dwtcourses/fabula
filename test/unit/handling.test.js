import { writeFileSync, readFileSync } from 'fs'
import { spawnSync } from 'child_process'
import { resolve } from 'path'

const 
  opts = { cwd: resolve(__dirname, '../fixtures') },
  fabulaBin = resolve(__dirname, '../../bin/fabula.js'),
  spawnFabula = (...args) => spawnSync(fabulaBin, args, opts)

describe('test handling', () => {

  test('single handler', async () => {
    const { stdout, stderr } = spawnFabula('handler')
    console.log(stdout)
  })
})
