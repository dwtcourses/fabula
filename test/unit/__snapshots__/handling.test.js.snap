// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test handling single handler 1`] = `
"[INFO] [local] [OK] write /tmp/fabula-handler-test:
[INFO] [local] 1
[INFO] [local] [OK] cat /tmp/fabula-handler-test
[INFO] [local] [OK] rm /tmp/fabula-handler-test
"
`;
